src/helm/benchmark/run_expander.py

```diff
@@ -308,6 +308,7 @@ class ModelRunExpander(ReplaceValueRunExpander):
         "openai/curie": ["openai/curie"],
         "chat_run": ["openai/chat-gpt", "openai/text-davinci-003"],  # Compare ChatGPT to text-davinci-003
         "all": get_all_models(),
+        "all_language": get_all_text_models() + get_all_code_models(),
```

Comment:
this might be confusing - i think one could reasonable think "language" just means "text". Maybe "string" or something is better, idk. Also what does this include that "all" doesnt?

Reply:
`all` will include image models